### PR TITLE
Minor message fixes

### PR DESCRIPTION
**Pull request by logical-1985516**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/StatusCommand.java [hunk 1]
```diff
@@ -30,8 +31,8 @@ public class StatusCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates recruitment status of a candidate"
             + " existing in the address book to one of the five status:\n"
             + "PRESCREEN, IN_PROGRESS, WAITLIST, ACCEPTED, REJECTED\n"
-            + "Parameters: INDEX(must be a positive integer) STATUS \n"
-            + "Example: " + COMMAND_WORD + " 2 ACCEPTED";
+            + "Parameters: INDEX (must be a positive integer) " + PREFIX_STATUS + "STATUS \n"
+            + "Example: " + COMMAND_WORD + " 2 " + PREFIX_STATUS + "ACCEPTED";
```
Comment (leeweiming3) [inline src/main/java/seedu/address/logic/commands/StatusCommand.java:35]: this doesn't match the actual behaviour of the command

File: src/main/java/seedu/address/logic/commands/TagCommand.java [hunk 2]
```diff
@@ -29,7 +29,7 @@ public class TagCommand extends Command {
             + "Existing tags will not be overwritten by the input.\n"
             + "At least one tag must be present.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_TAG + "[tag]...\n"
+            + PREFIX_TAG + "[TAG]...\n"
```
Comment (leeweiming3) [inline src/main/java/seedu/address/logic/commands/TagCommand.java:32]: [] means optional. Should it be on the whole "t/TAG"? (technically from my code)

**Review by leeweiming3 — DISMISSED** [review]
The status command doesn't match the new description. Other than that, looks good. Consider adding a space to the edit command message that is missing.

File: src/main/java/seedu/address/logic/commands/TagCommand.java (hunk 2 above)
Comment (acekhoon) [reply src/main/java/seedu/address/logic/commands/TagCommand.java:32]: Yep I agree with Wei Ming

**Review by acekhoon — CHANGES_REQUESTED** [review]

**Review by acekhoon — APPROVED** [review]
LGTM

**Review by acekhoon — APPROVED** [review]
LGTM

**Pull request merged**